to_blockchain/data_storage_contract.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity ^0.4.23;
+
+contract data_storage_contract {
+    
+    struct Freelancer {
+        string id; //freelancer id hash
+        string name; //freelancer's name
+        address projectAdress; //project's ethereum address
+        uint32 hoursSpent; 
+        uint32 tasksCompleted; 
+    }
+
+    // struct for storing projects related information
+    struct Project {  
+        string id;  // project id hash
```

Comment:
UUID are 16 bytes (128 bits) numbers and should probably be stored as this for efficiency-sake. so uint128.

Conversion example:

```
SELECT ('x' || translate(left('47715f86-af1c-411b-b982-726c850d9b95'::text, 18), '-', ''))::bit(64)::bigint;
```